wordpress-mobile/WordPress-iOS
# Today Widget - Tracks

Fixes #15158

To test:
Pre requisites:
- Better to start from scratch and delete any previously installed version if you had tested widgets before.
- Make sure you have iOS 14.3 installed on your device
- In the developer settings, enable "WidgetKit Developer Mode"

1. Build and run this version
2. Install one or more widgets
3. In mission control, select the tracks live view tool (Metrics -> Tracks -> Live view), and input `wpios_today_home_extension_widget_updated` as event name
4. Wait for at least five minutes (sometimes it takes 8/9 minutes) and see the incoming tracks in the log
5. Expand the tracks and verify that the info medium_widgets, small_widgets and total_widgets are consistent with that you installed

Known issue: two widgets of the same size and the same site, seem to be counted as one.

PR submission checklist:
